x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/configure_datasource.tsx

```diff
@@ -35,9 +35,12 @@ export const ConfigureEndpointDatasource = memo<CustomConfigureDatasourceContent
             <p>
               {from === 'edit' ? (
                 <LinkToApp
-                  appId="securitySolution"
+                  data-test-subj="editLinkToPolicyDetails"
+                  appId="securitySolution:management"
                   appPath={policyUrl}
-                  href={`${services.application.getUrlForApp('securitySolution')}${policyUrl}`}
+                  href={`${services.application.getUrlForApp(
```

Comment:
After some investigation by @parkiino we found that we really can't use the `formatUrl()` method provided by the Security App's hook because that hook also calls another hook that requires Redux - and since this component is used from app external to the Security App that does not have redux, it fails. 
Closing this out :)